ihontaryk/homework_10/context_manager_try_except.py

```diff
@@ -0,0 +1,28 @@
+class Catcher:
+    def __init__(self, *exceptions):
+        self.exceptions = exceptions or Exception
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if exc_type:
+            return issubclass(exc_type, self.exceptions)
+
+
+def divide_numbers(num1, num2):
+    return num1/num2
```

Comment:
do not forget about spaces around operators
use flake8 from time to time

return num1 / num2